utils/src/test/java/org/apache/cloudstack/utils/hypervisor/HypervisorUtilsTest.java

```diff
@@ -106,6 +104,8 @@ private void setupcheckVolumeFileForActivityFile(File file, long minSize) throws
             writer.write(chars);
             written += chars.length;
         }
+        long creationTime = System.currentTimeMillis();
```

Comment:
Not really, if you move it one line down, after <code>writer.close()</code>, the test could still give false positives because there could still be a hickup during <code>writer.close()</code> while the file has already been written.